person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(_id, _age, name = 'unknown')
```

Comment:
- [ ] The Person class should have a **parent_permission** as optional and have a default value of **true** as requested by the previous reviewer. :blush: